### DISCUSSION TITLE
Where is the apk to download?

### DISCUSSION THREAD

**Discussion by GhettoBluez** [skyline-emu/skyline#1438] [General]
Hi guys and thank you for your awesome work! I am wondering where is the apk file to try it for myself in my Poco F3 smartphone?

Also, I want to send a donation. Can you please tell me how I do it?  Thanks to all!

**Comment by Crytonics** [comment]
You can download apk from github tab Actions or here https://download.skyline-emu.one/

**Comment by PixelyIon** [comment]
Additionally, we do not have any way to donate at the moment but we do appreciate your willingness to do so. We will likely have this in the future, it will be announced everywhere once it's available.

**Comment by GhettoBluez** [comment]
Ok. Thank you very much. I will stay tuned!
